### ISSUE TITLE
feature: filter out architectures for openSUSE repos

### ISSUE THREAD

**Issue by jonsger** [SUSE/rmt#906]
It would be nice if one could "filter out" specific architectures from openSUSE repos. This is currently not possible because openSUSE has only one repo for all CPU architectures.

You can save quite same disk space and download bandwidth then.

**Comment by doccaz** [comment]
That would be wonderful. I'm currently struggling with getting thousands of unnecessary packages downloaded on my customer's RMT.

**Comment by JPrazak** [comment]
I have same issue:

Product Leap 15.4  > selecting any product activate all and deselecting one deactivate all arch
| 2406 | openSUSE Leap     | 15.4    | aarch64 | Don't Mirror | 
| 2407 | openSUSE Leap     | 15.4    | ppc64le | Don't Mirror |
| 2408 | openSUSE Leap     | 15.4    | s390x   | Don't Mirror |
| 2409 | openSUSE Leap     | 15.4    | x86_64  | Don't Mirror | 

Repos for Leap 15.4  >  not possible to chose an arch 
| 5593 | openSUSE-Leap-15.4-NonOss-Pool          | Not Mandatory | Don't Mirror |
| 5589 | openSUSE-Leap-15.4-NonOss-Updates   | Not Mandatory | Don't Mirror |
| 5592 | openSUSE-Leap-15.4-Pool                       | Mandatory   |
| 5588 | openSUSE-Leap-15.4-Updates                 | Mandatory   |

**Comment by ramzcode** [comment]
Anyone found solution for this ?

**Comment by doccaz** [comment]
Nope, it still downloads everything as the architectures are under the same folder and sharing the same repodata... See https://download.opensuse.org/distribution/leap/15.4/repo/oss/ for example.

**Comment by DoctorCr4nk** [comment]
Bump - I'm facing the same issue, would love to configure custom repos so they only mirror the architectures I specify
